internal/strings/strings.go

```diff
@@ -98,6 +98,11 @@ func MaybeUnquote(s string) string {
 	res := strings.Builder{}
 	tail := s[1 : len(s)-1]
 	for len(tail) > 0 {
+		if len(tail) >= 4 && tail[:4] == `\x5c` {
```

Comment:
Please do link the reasoning for this and also what are you trying to overcome because it isn't obvious.